docs/flashing_preparation.md

```diff
@@ -1,14 +1,5 @@
 # Flashing sample applications
 
```

Comment:
This document lacks information about the required connection to the power supply of the nRF52840-DK during flashing

Reply:
Added.